### PR TITLE
Update 06_how-do-i-put-intro-text-about-myself-on-my-profile.md

### PR DESCRIPTION
**Pull request by sylee-**

### CODE REVIEW

**Review by emfet — APPROVED** [review]
👍

File: 02_profiles/06_how-do-i-put-intro-text-about-myself-on-my-profile.md [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
+3. Select "(edit)" and start typing!
+4. Once you have everything the way you want it, submit the form, and just like magic, the text will appear on your profile.
+
+For the third method, follow these steps to add to or edit your About Me section.
+1. Select the hamburger menu (the icon with three horizontal lines) at the top right-hand corner of the page.
+2. Select "Edit Profile" in the dropdown menu.
+3. Select the tab titled "About You” and get busy typing!
+4. Once you have everything the way you want it, submit the form, and presto, the text will appear on your profile.
+
+Lather, rinse, and repeat as many times as you’d like until you’re satisfied.
```
Comment (DarklingMuse) [inline 02_profiles/06_how-do-i-put-intro-text-about-myself-on-my-profile.md:21]: I'd add an exclamation mark:  Lather, rinse, and repeat as many times as you’d like until you’re satisfied!

Comment (sylee-) [reply 02_profiles/06_how-do-i-put-intro-text-about-myself-on-my-profile.md]: Done.

**Review by DarklingMuse — APPROVED** [review]
👍

**Comment by DarklingMuse** [comment]
New Suggestion (Not editing this directly to avoid conflicts :P)

File: 02_profiles/06_how-do-i-put-intro-text-about-myself-on-my-profile.md [hunk 2]
```diff
@@ -1,12 +1,10 @@
-# How do I put intro text about myself on my profile?
+# How do I edit the About Me section of my profile?
```
Comment (nielsenramon) [inline 02_profiles/06_how-do-i-put-intro-text-about-myself-on-my-profile.md:1]: `... how do I edit the about section of ...`

File: 02_profiles/06_how-do-i-put-intro-text-about-myself-on-my-profile.md [hunk 3]
```diff
@@ -1,12 +1,10 @@
...
 
-You know people want to know about you - come on, who wouldn't?
+You can edit the About Me section of your profile at any time by:
```
Comment (nielsenramon) [inline 02_profiles/06_how-do-i-put-intro-text-about-myself-on-my-profile.md:3]: `... the About section ...`

File: 02_profiles/06_how-do-i-put-intro-text-about-myself-on-my-profile.md [hunk 4]
```diff
@@ -1,12 +1,10 @@
...
 
-To add text to your profile:
+- Going to [your profile settings](https://fetlife.com/settings/profile/about)
```
Comment (nielsenramon) [inline 02_profiles/06_how-do-i-put-intro-text-about-myself-on-my-profile.md:5]: `Visiting https://fetlife.com/settings/profile/about`

File: 02_profiles/06_how-do-i-put-intro-text-about-myself-on-my-profile.md [hunk 5]
```diff
@@ -1,12 +1,10 @@
...
+- Selecting the About Me tab on the left side of the page
```
Comment (nielsenramon) [inline 02_profiles/06_how-do-i-put-intro-text-about-myself-on-my-profile.md:6]: Remove

File: 02_profiles/06_how-do-i-put-intro-text-about-myself-on-my-profile.md [hunk 6]
```diff
@@ -1,12 +1,10 @@
...
+- Editing the text
```
Comment (nielsenramon) [inline 02_profiles/06_how-do-i-put-intro-text-about-myself-on-my-profile.md:7]: Filling out and submitting the form

File: 02_profiles/06_how-do-i-put-intro-text-about-myself-on-my-profile.md [hunk 7]
```diff
@@ -1,12 +1,10 @@
...
+- Submitting the form
```
Comment (nielsenramon) [inline 02_profiles/06_how-do-i-put-intro-text-about-myself-on-my-profile.md:8]: Remove

**Review by nielsenramon — APPROVED** [review]

**Pull request merged**